### PR TITLE
Remove dependencies to splinter/selenium and all related tests

### PR DESCRIPTION
**Pull request by cedricmessiant**
+ update the docs to remove references to splinter/selenium
Closes https://bluenove.atlassian.net/browse/DEVAS-1854

### CODE REVIEW

**Comment by maparent** [comment]
it's complaining about pytest-localserver, I guess you'd have to remove that in the code as well.

File: requirements-dev.frozen.txt [hunk 1]
```diff
@@ -19,7 +18,6 @@ alabaster==0.7.10         # via sphinx
 alembic==0.8.6
 amqp==2.3.2               # via kombu
 anyjson==0.3.3
-appnope==0.1.0            # via ipython
```
[the remark this answers is not in the document]
Comment (cedricmessiant) [reply requirements-dev.frozen.txt]: I have no idea as I said in the channel. It is a mac os thing if I understood well so it should not be in requirements-prod. Do you want me to try to reinstall ipython and see what happens?

[the remark this answers is not in the document]
Comment (cedricmessiant) [reply requirements-dev.frozen.txt]: I still have `ipython` that works

File: requirements-dev.frozen.txt [hunk 2]
```diff
@@ -179,13 +173,11 @@ social-auth-storage-sqlalchemy==1.1.0
 sphinx==1.5.1
 sphinxcontrib-httpdomain==1.5.0
 sphinxcontrib-zopeext==0.2.1
-splinter==0.7.6
 sqlalchemy-schemadisplay==1.3
 sqlalchemy==1.2.2
 sqlparse==0.2.4
 structlog==17.2.0
 supervisor==3.3.0
-termcolor==1.1.0
```
[the remark this answers is not in the document]
Comment (cedricmessiant) [reply requirements-dev.frozen.txt]: this one is not in `ipython` deps (it has been removed when I removed splinter and selenium)

File: doc/testing.rst [hunk 3]
```diff
@@ -327,5 +327,4 @@ TODO
 .. _TestRequest: http://docs.pylonsproject.org/projects/webtest/en/latest/api.html#webtest-app-testrequest
 .. _WebTest: http://docs.pylonsproject.org/projects/webtest/en/latest/
 .. _`Pyramid Documentation`: http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/testing.html
-.. _Splinter: https://splinter.readthedocs.io/en/latest/
```
[the remark this answers is not in the document]
Comment (cedricmessiant) [reply doc/testing.rst]: yeah, thanks

**Pull request merged**